doc/source/_templates/layout.html

```diff
@@ -2,53 +2,12 @@
 
 {%- block extrahead %}
 <style>
-  .main {
-    -moz-box-shadow: none;
-    -webkit-box-shadow: none;
-    box-shadow: none;
-  }
-  div.top-scipy-org-logo-header {
-    background-color: #fafafa;
-    border-bottom: 2px solid #013243; /* Warm Black */
-    margin-top: 0;
-    box-shadow: none;
-  }
-  div.top-scipy-org-logo-header img {
-    height: 100px;
-    padding-left: 27px;
-  }
-  div.spc-navbar .nav-pills > li > a {
-    background-color: #4d77cf; /* Han Blue */
-  }
+.navbar-brand img {
+   height: 75px;
+}
+.navbar-brand {
+   height: 75px;
+}
```

Comment:
At this point, it might make more sense to put this in the `html_css_files` `conf.py` setting, rather than use the template.

Reply:
Indeed, if the css is the only think being changed to the layout.html, I think it's more logical to do this with a custom css file

Reply:
Can we leave that for another PR once things settle down a bit?

Reply:
Sure, I suppose you can argue that CSS being in this file was a pre-existing issue.